src/Resources/config/routing.yml

```diff
@@ -19,11 +19,46 @@ bitbag_sylius_wishlist_plugin_shop_wishlist_add_product_variant:
         _controller: bitbag_sylius_wishlist_plugin.controller.action.add_product_variant_to_wishlist
 
 bitbag_sylius_wishlist_plugin_shop_wishlist_remove_product_variant:
-    path: /wishlist/remove/variant/{variantId}
+    path: /wishlist/{wishlistId}/remove/variant/{variantId}
     defaults:
         _controller: bitbag_sylius_wishlist_plugin.controller.action.remove_product_variant_from_wishlist
 
 bitbag_sylius_wishlist_plugin_shop_wishlist_render_header_template:
     path: /wishlist/header-template
     defaults:
         _controller: bitbag_sylius_wishlist_plugin.controller.action.render_header_template
+
+bitbag_sylius_wishlist_plugin_shop_wishlist_clean:
+    path: /wishlist/clear/{wishlistId}
+    defaults:
+        _controller: bitbag_sylius_wishlist_plugin.controller.action.clean_wishlist
+
+bitbag_sylius_wishlist_plugin_shop_wishlist_create_new_wishlist:
+    path: /createWishlist
```

Comment:
1. As above, Sylius Resource,
2. Keep the naming consistent; below you've `wishlists/`, keep it here.